packages/mdc-fab/README.md

```diff
@@ -39,68 +37,71 @@ npm install --save @material/fab
 
 ## Usage
 
-The demonstrations use the [Material Design Icon Font](https://design.google.com/icons/).
-You may include this to use them as shown or use any other icon method you wish.
+### Load Material Icons
 
-### Default
+We recommend you load [Material Icons]((https://design.google.com/icons/)) from Google Fonts
 
 ```html
-<button class="mdc-fab material-icons" aria-label="Favorite">
-  <span class="mdc-fab__icon">
-    favorite
-  </span>
-</button>
+<head>
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
 ```
 
-### Mini
+### HTML Structure
 
 ```html
-<button class="mdc-fab mdc-fab--mini material-icons" aria-label="Favorite">
+<button class="mdc-fab material-icons" aria-label="Foo label">
   <span class="mdc-fab__icon">
-    favorite
+    foo_icon
```

Comment:
Is there a reason not to use a valid material-icons value here? (We also still reference `favorite` in the absolute-positioning example.)